datahub-web-react/src/app/entity/shared/constants.ts

```diff
@@ -47,4 +47,12 @@ export const EMPTY_MESSAGES = {
         title: 'No domain set',
         description: 'Group related entities based on your organizational structure using by adding them to a Domain.',
     },
+    contains: {
+        title: 'No contain yet',
+        description: 'Share your knowledge by adding contain.',
+    },
+    inherits: {
+        title: 'No inherit yet',
```

Comment:
Does not inherit from any terms 

Terms can inherit from other terms to represent an "Is A" style relationship